chalicelib/action.py

```diff
@@ -350,3 +347,24 @@ def _valid_reason(self, reason: str) -> bool:
             return True
 
         return False
+
+    def _create_list_message(self, data) -> None:
+        """
+        Create the slack block message layout for list action
+        """
+        data = json.loads(data)
```

Comment:
Just a thought; Do you think it's better that this method expects a native data type instead of json?

Reply:
In the end I think the blocks are json.. so it is fine with json actually makes it easier.